src/Me/components/FormField/FormField.js

```diff
@@ -3,24 +3,43 @@ import styled from 'styled-components';
 import uniqueId from 'lodash/uniqueId';
 import { formFieldContext } from './formContext';
 
+const InlineHelpText = styled.span`
+  color: #5c5c5c;
+  padding-left: 5px;
+  font-weight: normal;
+
+  &:before {
+    content: '(';
+  }
+
+  &:after {
+    content: ')';
+  }
+`;
+
 const FormFieldContainer = styled.div`
   display: flex;
   flex-direction: column;
   font-family: Lato, sans-serif;
   margin-bottom: 30px;
 `;
+
 const Label = styled.label`
   font-weight: bold;
   font-size: 14px;
   line-height: 17px;
   margin-bottom: 6px;
 `;
 
-export const FormField = ({ label, className, children }) => {
+export const FormField = ({ label, className, children, helpText }) => {
   const [id] = useState(() => uniqueId('form-field-'));
   return (
     <FormFieldContainer className={className}>
-      {label && <Label htmlFor={id}>{label}</Label>}
+      {label && (
+        <Label htmlFor={id}>
+          {label}{helpText && <InlineHelpText>{helpText}</InlineHelpText>}
```

Comment:
@moshfeu  There is not actaul space between `label` and `helpText`. I understand there is a `padding-left` on `InlineHelpText` but if user double click on the text to select it or some accecability software is trying to read the text, it will read it as one word.
I might be wrong about it but I figure it might be worth mentioning